### ISSUE TITLE
Train Script or Pretrain Models

### ISSUE THREAD

**Issue by HanshengGUO** [chenpan0615/FCCDN_pytorch#7]
Hi,

Thanks for your great work. I'm trying to train the model on Levir-CD but cannot get the reported result. Would you please provide some pretrained models or train scipt for me to train the model?

Thanks again.

**Comment by chenpan0615** [comment]
Of course, I can provide the pretrained model. It will take several days to retrain the model since I have deleted all the checkpoints due to the limited storage.

**Comment by HanshengGUO** [comment]
Thanks again for your prompt response. Looking forward to updates from you!

**Comment by chenpan0615** [comment]
Hi, I have uploaded the pretrained weights along with the inferecne script.  Please note that the weights are trained with another version of code. The new code is slightly different from the old version (changed the upsample mode from 'bilinear' to 'nearest'). This difference is not suppose to affect the performance. So, if you want to inference with  the pretrained model, please make sure you have also updated the implementation of network.

**Comment by HanshengGUO** [comment]
Hi, Thanks again for your wonderful work. I'm trying to implement it but still cannot reach the reported F1. Would you please also provide the train script? If not, can you give more information on the training detail? Like, do you use any data augmentation or model soups, etc?

**Comment by chenpan0615** [comment]
I'm afriad I can not provide the training code. It is a part of our teamwork and is not going to open source recentlty. I do use data augmentation strategies and the details are presented in the paper. I was wondering what score can you get with my code? Have you ever try to augment both pretemporal and posttemporal image? I also use a warm-up strategy by increasing the lr from 1e-7 to 2e-3 in the first 200 iterations. In addition, you can also try to set a bigger lr paticence since in some cases the lr reduced earlier than expected.

**Comment by HanshengGUO** [comment]
Got it. Thanks for your information and I'll try them first.
